src/Pages/Auth/Login.jsx

```diff
@@ -4,12 +4,10 @@ import { Navigate } from 'react-router-dom';
 import UserLogin from '../../Api/ApiCalls';
 
 function Login() {
-  const [FormData, setFormData] = useState({
-    email: '',
-    password: '',
-  });
+  const [FormData, setFormData] = useState({});
+  const [error, setError] = useState(null);
+  const [Loading, setLoading] = useState(false);
```

Comment:
[Optional] Using a specific naming convention can be very helpful for the code clarity, and I recommend to follow some basic naming convention rules. For example it's better to use the camelCase naming system for our variables and in this example it can be `loading` instead of `Loading`. This rule can be applied on all other variables that I won't mention here.

Reply:
@aimalamiri suggestion is well noted 👍